packages/paste-website/src/pages/customization/customizing-component-elements.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Customizing Component Elements
+description: A powerful concept for any programmable application is extending existing components and customizing their look and feel.
+slug: /customization/paste-for-programmable-apps/
+---
+
+import {graphql} from 'gatsby';
+import {ProgrammableAppsCallout} from '../../components/page-components/customization/customization-provider/ProgrammableAppsCallout';
+import {P} from '../../components/Typography';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/customization/customizing-component-elements/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<P variant="lead">{props.pageContext.frontmatter.description}</P>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+<ProgrammableAppsCallout />
+
+We are currently working on this feature and providing support for it gradually across the entire suite of components. Watch this space!
```

Comment:
This seems like a prime opportunity to collect emails. Like, sign up for this announcement + more. Can make a re-usable CTA component for places like this.

Reply:
Mmmmm, I like that. I can create a ticket for a sign up form